vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -248,6 +262,10 @@ public void saveAccount(@Nullable VimeoAccount vimeoAccount, String email) {
         mConfiguration.saveAccount(vimeoAccount, email);
     }
 
+    public void setConfiguration(@NotNull Configuration configuration) {
```

Comment:
I guess new public method should probably include javadoc.

Reply:
I removed this method I had added. In the current implementation, I think the client should create the configuration object through the builder and only provide the object through the initialize method. There should be one way to do so.